### PR TITLE
Support new session queries

### PR DESCRIPTION
**Pull request by SpennyNDaJets**
## Summary
The new query logic requires new endpoints that support the new language. Create these new endpoints and respective functions to process the new query language.

Unrelated changes:
- Deprecate old query language functions and endpoints
- Capitalize table configs

Follow ups:
- Apply sampling for sessions

## How did you test this change?
Confirm sessions and histogram are still working

## Are there any deployment considerations?
N/A - not being used in production

## Does this work require review from our design team?
N/A

### CODE REVIEW

File: backend/stepfunctions/client.go [hunk 1]
```diff
@@ -12,13 +12,15 @@ import (
...
+	utilsV2 "github.com/highlight-run/highlight/backend/lambda-functions/deleteSessionsV2/utils"
 	utils2 "github.com/highlight-run/highlight/backend/lambda-functions/sessionExport/utils"
 	"github.com/highlight-run/highlight/backend/model"
 )
 
 var (
-	deleteSessionsArn = os.Getenv("DELETE_SESSIONS_ARN")
-	sessionExportArn  = "arn:aws:states:us-east-2:173971919437:stateMachine:SessionExport"
+	deleteSessionsArn   = os.Getenv("DELETE_SESSIONS_ARN")
+	deleteSessionsV2Arn = os.Getenv("DELETE_SESSIONS_V2_ARN")
```
Comment (SpennyNDaJets) [inline backend/stepfunctions/client.go:22]: TODO(spenny): add this when deployed

File: backend/clickhouse/sessions.go [hunk 2]
```diff
@@ -589,3 +590,108 @@ func (client *Client) SessionsKeyValues(ctx context.Context, projectID int, keyN
...
+	if limit != nil {
+		sb = sb.Limit(*limit)
+	}
+	if offset != nil {
+		sb = sb.Offset(*offset)
+	}
+
+	sql, args := sb.BuildWithFlavor(sqlbuilder.ClickHouse)
+
+	// TODO(spenny): custom rules to check - sampleField
```
Comment (SpennyNDaJets) [inline backend/clickhouse/sessions.go:618]: TODO(spenny): figure out with Julian the best way to add a new sample with the new search UI, and apply here

**Review by mayberryzane — APPROVED** [review]
for the lambda functions, were you going to create separate ones in AWS? do we have to add those to the `deploy to AWS` tasks?

**Comment by SpennyNDaJets** [comment]
> for the lambda functions, were you going to create separate ones in AWS?

From our convo in stand up, we will change this lambda to use the new logic once verified, rather than creating a new lambda

**Pull request merged**